sidebars.ts

```diff
@@ -281,6 +281,7 @@ const sidebars: SidebarsConfig = {
                 'platform/howto/byoc/create-custom-cloud',
                 'platform/howto/byoc/assign-project-custom-cloud',
                 'platform/howto/byoc/add-customer-info-custom-cloud',
+                'platform/howto/byoc/tag-custom-cloud-resources',
```

Comment:
Should we have `rename-custom-cloud` and  `delete-custom-cloud` next to one another?